NetKet/Machine/pylayer.hpp

```diff
@@ -65,6 +65,22 @@ void AddLayerModule(py::module &m) {
                                                                    "SumOutput")
         .def(py::init<int>(), py::arg("input_size")) ADDLAYERMETHODS(LayerType);
   }
+  {
+    using LayerType = Activation<MachineType, Lncosh>;
+    py::class_<LayerType, AbLayerType, std::shared_ptr<LayerType>>(subm,
+                                                                   "Lncosh")
+        .def(py::init<int>(), py::arg("input_size")) ADDLAYERMETHODS(LayerType);
+  }
+  {
+    using LayerType = Activation<MachineType, Tanh>;
+    py::class_<LayerType, AbLayerType, std::shared_ptr<LayerType>>(subm, "Tanh")
+        .def(py::init<int>(), py::arg("input_size")) ADDLAYERMETHODS(LayerType);
+  }
+  {
+    using LayerType = Activation<MachineType, Relu>;
+    py::class_<LayerType, AbLayerType, std::shared_ptr<LayerType>>(subm, "Relu")
+        .def(py::init<int>(), py::arg("input_size")) ADDLAYERMETHODS(LayerType);
+  }
 }
 
 }  // namespace netket
```

Comment:
I see, the thing is that you didn't remove the activation module `pyactivation.hpp`. Is it because you want to do changes to the Conv and FullyConn layers so that they do not take activation as an input parameter anymore?

Reply:
I didnt want to break any of the old stuff. I wanted to do a minimal additive change first.
But sure we can decide now if want to no longer have the activation within the layer.
This would require some adjustments to the lookup

Reply:
OK, but eventually the idea would be to separate convolutions and fully connected layers from activations I guess... like in Keras 
Do you need adjustements in the quantity stored in the look up or more complicated changes?

Reply:
no it would a simplification actually

Reply:
Yeah I figured this would simplify things , that's cool

Reply:
Ok let's merge this and then we can deal with the other changes to the layers later, including the convnet bug